Source/WTF/wtf/fast_float/ascii_number.h

```diff
@@ -1,6 +1,7 @@
 #ifndef FASTFLOAT_ASCII_NUMBER_H
 #define FASTFLOAT_ASCII_NUMBER_H
 
+#include <bit>
```

Comment:
This is third party code - it will be undone when we sync again, should we do it upstream?

Reply:
The header is using `WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN` so it's not really third-party anymore and it already cannot be re-sync'd without breaking the build.